CNAME

```diff
@@ -1 +1 @@
-nodeschool.cz
+nodeschool.cz
```

Comment:
This change seems unrelated to me, can you revert it please?

Reply:
reverted